libgalois/include/katana/PODVector.h

```diff
@@ -71,25 +76,44 @@ class PODVector {
   typedef std::reverse_iterator<iterator> reverse_iterator;
   typedef std::reverse_iterator<const_iterator> const_reverse_iterator;
 
-  PODVector() : data_(NULL), capacity_(0), size_(0) {}
```

Comment:
You are removing the default constructor. I think the compiler might be generating one, but it might be better to be explicit. I think (though I'm not sure) that an explicit constructor which can be called with zero-arguments via a default parameter is *different* from a default constructor (not marked explicit and with zero-arguments natively).

Reply:
I've tested and such constructors seem to behave exactly the same. Without the no-argument constructor, the code compiles well, including the places that use `PODVector` with default construction (e.g. inside an `std::vector`). However, if I add an explicit no-argument constructor, I get compilation errors: `error: call of overloaded ‘PODVector()’ is ambiguous`. So there should be no problem here with the code as it is.